abstracts/bigtable.md

```diff
@@ -8,4 +8,5 @@ processing to real-time data serving). Despite these varied demands, Bigtable
 has successfully provided a flexible, high-performance solution for all of these
 Google products. In this paper we describe the simple data model provided by
 Bigtable, which gives clients dynamic control over data layout and format, and
-we describe the design and implementation of Bigtable.
+we describe the design and implementation of Bigtable
+modification.
```

Comment:
Should this be on the previous line?

Reply:
Thanks for this suggestion! Should I start a new PR?

Reply:
Up to you since this repo is just for exploration. I can merge this as-is or, if you want, you can make another commit on top of the merge to address this and push, and we can merge that.